### PR TITLE
chore: prepare to implement snippetgen for standard call

### PR DESCRIPTION
**Pull request by dizcology**
Internal changes only, without effect on the generated code:

1. Add docstrings.
2. Replace internal utility method `_append_to_sample_function_def_body` with `_extend_sample_function_def_body `.  (But the low-level libcst implementation of the actual code modification remains "append" for now).
3. Add top-level switch for `call` type, in preparation to implement code generation for standard call.

### CODE REVIEW

**Review by amanda-tarafa — APPROVED** [review]
LGTM on SnippetGen side.

File: tests/unit/configurable_snippetgen/test_configured_snippet.py [hunk 1]
```diff
@@ -244,3 +276,8 @@ def test_code_without_endpoint(snippet_without_endpoint):
     client = speech_v1.AdaptationClient()
 """
     assert snippet_without_endpoint.code == expected_code
+
+
+def test_generate_should_raise_error_if_unsopported(snippet_bidi_streaming):
```
Comment (parthea) [inline tests/unit/configurable_snippetgen/test_configured_snippet.py:281]:
```suggestion
def test_generate_should_raise_error_if_unsupported(snippet_bidi_streaming):
```

**Review by parthea — APPROVED** [review]

**Comment by vchudnov-g** [comment]
@dizcology OK with suggestion?

File: tests/unit/configurable_snippetgen/test_configured_snippet.py (hunk 1 above)
Comment (dizcology) [reply tests/unit/configurable_snippetgen/test_configured_snippet.py]: Thanks for catching this!

**Comment by dizcology** [comment]
Thanks @vchudnov-g, committed.  I will merge this once the checks pass.

**Pull request merged**